src/components/events/EventPosterUpload.tsx

```
@@ -79,9 +109,46 @@ export const EventPosterUpload: React.FC<EventPosterUploadProps> = ({ eventId, o
 
     return (
         <div className="space-y-4">
+            <div className="space-y-1">
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)